examples/reagent/src/cljs/todomvc/actions.cljs

```diff
@@ -0,0 +1,39 @@
+(ns todomvc.actions
+  (:require [todomvc.session :as session]
```

Comment:
Please ignore my last comment.  After some more thought, to answer your original question:

*Is the spaces indentation a requirement? All our code should use tabs if that's possible.*

I believe the answer is yes, spaces indentation is a requirement. The following recommend against tabs for clojure:
* https://github.com/bbatsov/clojure-style-guide
* https://clojuredocs.org/examples-styleguide
* http://en.wikibooks.org/wiki/Learning_Clojure/Coding_Conventions

As a result, I will untab `project.clj`.  My apologies for not originally thinking this through.